dlp/jobs_test.py

```diff
@@ -59,23 +59,40 @@ def test_job_name():
         print("Issue during teardown, missing job")
 
 
+def retry_on_assertion_error(exception):
+    return isinstance(exception, (AssertionError))
+
+
 def test_list_dlp_jobs(test_job_name, capsys):
-    jobs.list_dlp_jobs(GCLOUD_PROJECT)
+    @retry(
```

Comment:
Does this not work directly on `test_list_dlp_jobs`?

Reply:
I doubt it. I got a feeling that pytest needs a plugin for decorator to work with pytest (think that `@flaky` doesn't work with pytest, you need to use `@pytest.mark.flaky`).

Reply:
Of course, we can use `@pytest.mark.flaky`, but I'd stick with `@retry` because of a semantic reason. But technically I can use `@pytest.mark.flaky` too.